src/RollbarClient.mts

```diff
@@ -0,0 +1,50 @@
+// Internal Imports
```

Comment:
I did the same `git mv` command for all these file renames, but unfortunately some of them show up as deletes then creates :(